frontend/app/services/EventbriteService.scala

```diff
@@ -103,6 +103,14 @@ abstract class LiveService extends EventbriteService {
   val gridService = GridService(Config.gridConfig.url)
   val contentApiService = GuardianContentService
 
+  def mkRichEvent(event: EBEvent): Future[RichEvent] = for {
```

Comment:
It was bugging me that we had the same `mkRichEvent` code (apart from the type) duplicated in the Live and Local services, so this is an attempt to move it up into the shared service. I'd appreciate some :eyes: on this as I just guessed my way to this and I have no idea if it's idiomatic or not. The pattern matching is non-exhaustive too so idk, it compiled though…

@ostapneko @chrisjowen